contracts/cw721-base/src/execute.rs

```diff
@@ -77,16 +80,16 @@ where
 }
 
 // TODO pull this into some sort of trait extension??
-impl<'a, T, C, E, Q> Cw721Contract<'a, T, C, E, Q>
+impl<'a, T, E1, E2, C, Q> Cw721Contract<'a, T, E1, E2, C, Q>
```

Comment:
```suggestion
impl<'a, T, E1, E2, C, Q> Cw721Contract<'a, T, E1, E2, C, Q = Empty>
```

How about giving this the same default value as [CosmWasm does](https://docs.rs/cosmwasm-std/1.0.0/src/cosmwasm_std/deps.rs.html#26-30)?

Reply:
Yeah, I think it would be great adding default on its struct.